components/builder-web/app/reducers/projects.ts

```diff
@@ -36,6 +36,9 @@ export default function projects(state = initialState["projects"], action) {
                     setIn(["ui", "current", "loading"], false);
             }
             else {
+                if (action.payload.visibility !== "public") {
+                    action.payload.visibility = "private";
+                }
```

Comment:
Something to be aware of is that projects can also be "hidden", which will be visible to people that are authenticated.

Reply:
Yep. This might look a little weird at first, but the alternative would've been to sprinkle lots of `if it's hidden, that really means it's private from the user's perspective, so show the "private" icon|radio|whatever-other-thing here instead` all over the app -- and since the write APIs seem to want a value of "private" anyway, it made sense for the UI to treat hidden and private as synonymous.

Reply:
Terrific, thank you for clarifying.  I figured as long as you were aware of all 3 states and their meaning, we're good.

Reply:
And the GIF doesn't show it, but the private and hidden stuff shows up everywhere it should when I'm signed in as an origin member. Woo!